### ISSUE TITLE
Add a "show password" option on Source login

### ISSUE THREAD

**Issue by creviera** [freedomofpress/securedrop-client#275]
## Description

A source needs to type a long codename in order to log in. This can easily contain typos, maybe capslock is enabled, and currently there isn't a way to verify what is typed. The source interface login page should have a way to show/hide password (the eye icon is pretty standard now).

**Comment by ninavizz** [comment]
YES!!! This has been discussed in the past, just never formalized in an Issue. Thank you!! 😸  

The SVG code for those icons are also on the Content Inventory Issue for the Workstation Client.

**Comment by ninavizz** [comment]
Wait @creviera I just realized—this is in the wrong repo, if it's for the Source interface it should be in the regular SecureDrop repo (I know, duh, and you were working late—heehee).
